Modulefile

```diff
@@ -0,0 +1,12 @@
+name    'garethr-docker'
```

Comment:
Modulefile is no longer required, it's been superseded by the metadata.json file